matsuoharuki/pbl2_group2
# Feature/get spot

Description:
- Added a function getting a spot and its reviews by its spot_id.

To test this pull request:
- Start server.
```
docker-compose up
```
- In Postman, set Body:

```
 {
    "spotId": "spot_id1111"
}
```
- Send a request: GET http://localhost:3000/spot/search

Success condition:
- You can see the requested spot and its reviews in the bottom of Postman.
